src/api/app/jobs/update_package_meta_job.rb

```diff
@@ -1,4 +1,6 @@
 class UpdatePackageMetaJob < ApplicationJob
+  # NOTE: Its important that this job run in queue 'default' in order to avoid concurrency
+
```

Comment:
Because `self.job_queue` is not a part of ActiveJob or DelayedJob. Its our own implementation which calls self.job_queue and only for `CreateJob`s here:

https://github.com/evanrolfe/open-build-service/blob/fix%2Fpackage_dirty_jobs/src/api/app/models/event/base.rb#L159